magento/pwa-studio
# Singleton style tag

## Description
- Updated style-loader version from `~0.23.1` to `~2.0.0`.
- Use `injectType: singletonStyleTag` in production mode to avoid having a separate style tag for every css module.
- Use `injectType: styleTag` in development mode, because `singletonStyleTag` inject type does not support sourceMaps.

## Related Issue
Closes #2905 

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps
1. Serve venia-concept in development mode. Open the site and check the `style` tags under the `head` tag. The css modules are transformed into many style tags.
2. Serve venia-concept in production mode, Open the site and check the `style` tags under the `head` tag, The css modules are transformed into a single style tag.

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
